src/WindowsTracing/include/WindowsTracing/TracerListener.h

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef WINDOWS_TRACING_TRACER_LISTENER_H_
+#define WINDOWS_TRACING_TRACER_LISTENER_H_
+
+#include "capture.pb.h"
+
+namespace orbit_windows_tracing {
+
+class TracerListener {
+ public:
+  virtual ~TracerListener() = default;
+  virtual void OnSchedulingSlice(orbit_grpc_protos::SchedulingSlice scheduling_slice) = 0;
```

Comment:
Long term, we should probably have one listener for both, windows and linux tracing.

Reply:
My initial thought was to go for a single listener since to reuse code from the Linux OrbitService, but in the end I agree that it's cleaner not to.